src/bundle.c

```diff
@@ -2041,7 +2041,10 @@ gboolean check_bundle(const gchar *bundlename, RaucBundle **bundle, CheckBundleP
 		}
 		g_debug("Downloaded temp bundle to %s", ibundle->path);
 #else
-		g_warning("Mounting remote bundle not supported, recompile with -Dnetwork=true");
+		g_set_error(error, R_BUNDLE_ERROR, R_BUNDLE_ERROR_FORMAT,
+				"Mounting remote bundle not supported, recompile with -Dnetwork=true");
```

Comment:
I'd leave it up to you but touching this would be a great option to fix the message. "Mounting" is valid only for streaming, thus maybe it's better to just omit the term. And `-Dnetwork=true` is not sufficient for streaming which is probably what we would like to suggest here.

Thus maybe:

```suggestion
				"Remote bundle not supported, recompile with -Dstreaming=true");
```

Reply:
Perhaps "Remote bundle access not supported, …"?